app/models/subnet.rb

```diff
@@ -121,7 +121,7 @@ class Jail < ::Safemode::Jail
   def self.network_reorder(order_string = 'network')
     adapter = connection.adapter_name.downcase
     if adapter.starts_with?('postgresql')
-      reorder(order_string.sub('network', 'inet(network)'))
+      reorder(Arel.sql(order_string.sub('network', 'inet(network)')))
```

Comment:
This is user input and I don't like that we pass it directly to SQL. It's kind of filtered in the controller, so we're good. But better safe than sorry.

Something like this seems safer:

```suggestion
      if order_string ~= /ASC$/
         reorder(Arel::Nodes::NamedFunction.new('INET', [Subnet.arel_table[:network].asc]))
      else
         reorder(Arel::Nodes::NamedFunction.new('INET', [Subnet.arel_table[:network].desc]))
      end
```